### DISCUSSION TITLE
feature request: return model size

### DISCUSSION THREAD

**Discussion by simonmcnair** [mlfoundations/open_clip#832] [Ideas]
I have an 8gb GPU, at a high level I would like to skip over any models which won't fix in VRAM.  I can't see a function to return model size (even if only approximately).
This would be useful for me, and for anyone who doesn't want to d/l a model which they have no hope of running.
Cheers

**Comment by rom1504** [comment]
https://github.com/mlfoundations/open_clip/issues/224

Would be great yeah.

Most of the work is about writing scripts to automate collection of this data
